MixItUp.Base/MixerAPI/ChatClientWrapper.cs

```diff
@@ -151,6 +151,28 @@ public async Task Whisper(string username, string message, bool sendAsStreamer =
             }
         }
 
+        public async Task<ChatMessageEventModel> WhisperWithResponse(string username, string message, bool sendAsStreamer = false)
+        {
+            if (this.GetBotClient(sendAsStreamer) != null)
+            {
+                message = this.SplitLargeMessage(message, out string subMessage);
+
+                ChatMessageEventModel firstChatMessage = await this.RunAsync(this.GetBotClient(sendAsStreamer).WhisperWithResponse(username, message));
+
+                // Adding delay to prevent messages from arriving in wrong order
+                await Task.Delay(250);
+
+                if (!string.IsNullOrEmpty(subMessage))
+                {
+                    await this.WhisperWithResponse(username, subMessage, sendAsStreamer: sendAsStreamer);
```

Comment:
I do know this will "chop off" part of the sent whisper for the "large message" sent, but I think this is acceptable as it's only used to let the sender know they sent something.

Reply:
Good by me